docs/FAQ.md

```diff
@@ -160,8 +160,15 @@ curl --insecure https://127.0.0.1:28183/api/v1/session
 
 ### Can I import an existing wallet?
 
-Yes, it is possible to import an existing wallet.
-However, not yet via the web interface, but with a few manual steps via the command line.
+Yes, importing an existing wallet can be done via the web interface since [Jam `v0.1.6`][jam-v0-1-6].
+Just make sure you are running [JoinMarket `v0.9.10`][jm-v0-9-10] or later.
+
+[jam-v0-1-6]: https://github.com/joinmarket-webui/jam/releases/tag/v0.1.6
+[jm-v0-9-10]: https://github.com/JoinMarket-Org/joinmarket-clientserver/releases/tag/v0.9.10
+
```

Comment:
maybe add a line how to do this?
Where can the user find this?

Reply:
Done. Do you think it should contain additional information?

Reply:
Maybe as a follow-up, import wallet could have a section with detailed information imo, yes.
With all ins and outs.

Reply:
I think this is a good idea. However, the import process is quite straight forward, so I'd wait for someone to come and want to have it explained in more detail. Okay with you?

Thanks, brother :pray:

Reply:
Sure